src/lib.rs

```diff
@@ -58,6 +68,8 @@ impl Contract {
             authors: UnorderedMap::new(StorageKey::AuthorToAuthorPosts),
             communities: UnorderedMap::new(StorageKey::Communities),
             featured_communities: Vec::new(),
+            projects: UnorderedMap::new(StorageKey::Projects),
+            last_project_id: 0,
```

Comment:
Above this:
<img width="677" alt="image" src="https://github.com/near/neardevhub-contract/assets/13259400/a1811900-9bb6-4fab-bb7c-4ae9a2754ef3">

should initiate with `StateVersion::V7`